subprojects/libxmlb.wrap

```diff
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = libxmlb
 url = https://github.com/hughsie/libxmlb.git
-revision = 0.1.13
+revision = master
```

Comment:
Presumably you'll tag libxmlb1 soon so this should probably go to 0.1.14 when you do

Reply:
Right, indeed.

Reply:
I think for now we can just use `37887fd02f2062ca3ca5c0846c51ae32ed835af9` on master like the gcab one.